### ISSUE TITLE
Audio... wonky

### ISSUE THREAD

**Issue by jcotman** [uTox/uTox#686]
I am having trouble with the audio in uTox standalone client, Debian Testing, Intel audio.

The issue(s) is:

1 - During a 1 on 1 call, the other use cannot hear me.
2 - I cannot hear the other user
3 - EXCEPT when I go to Settings --> Audio and Video --> Preview and click the phone icon.  

In that case, the phone icon turns red and I hear the other user, but very choppily. He still cannot hear me.  It's possible that he could also hear me choppily if he also hit the "Preview" button during a chat, but I can't say one way or another.

I can verify that push to talk was not on, on my end.  Audio filtering WAS on.

Also, when I tried to "Preview" myself before making the call, the audio was very clear, albeit delayed by about 1.5 seconds.  But that's a different issue.

What information can I provide to make this report more useful?

Thank you. And thank you for uTox. I'm looking forward to using it.

**Comment by GrayHatter** [comment]
> What information can I provide to make this report more useful? 

Nothing, this bug report is awesome, and I'm pretty sure I know how to fix it.

> Also, when I tried to "Preview" myself before making the call, the audio was very clear, albeit delayed by about 1.5 seconds. But that's a different issue.

If you're talking about your audio, that's intentional, and the other side shouldn't have that delay (network delay should be MUCH less)

**Comment by jcotman** [comment]
> If you're talking about your audio, that's intentional, and the other side shouldn't have that delay (network delay should be MUCH less)

Correct. I am talking about *my* audio (hearing myself in my headphones through my mic) when referring to the 1.5 second delay during "Preview" when not on a call.  So that side-bug isn't a bug after all.  Excellent!
